angularjs-portal-home/src/main/resources/endpoint.properties.example

```diff
@@ -48,4 +48,4 @@ uwmadisonreddit.attributes=
 
 wiscdirectory.uri=http://www.wisc.edu/directories/json
 
-wiscedusearch.uri=http://ajax.googleapis.com/ajax/services/search/web
+wiscedusearch.uri=http://ajax.googleapis.com/ajax/services/search/web
```

Comment:
We like a new line at the end of each file. otherwise gh gives use that yucky x arrow thingy. Red bad.